src/background/certificate/providers/ServerProvider.ts

```diff
@@ -21,8 +34,7 @@ export class ServerProvider implements CertificateProvider {
     url: string
   ): Promise<RawCertificate> {
     return new Promise((resolve, reject) => {
-      console.log(url);
-      fetch(SERVER_URL + url)
+      fetch(this.serverUrl + url)
```

Comment:
- This does now fail very often since the user can adjust it. Currently the error message is: "Failed to execute 'fetch' on 'Window': Failed to parse URL from http://localhost:3000www.google.com" or "Failed to fetch" we should provide clear error messages and catch the case where the server url does not have a ending `/`

Reply:
Implemented.